src/galileo/experiments.py

```
@@ -83,6 +83,9 @@ def create_run_scorer_settings(
                     scorers.append(ScorerConfig.from_dict(scorer.to_dict()))
                     break
 
+        if not scorers:
+            raise ValueError(f"Non-existent metric is specified `{' '.join(metrics)}`")
```

Instead of outputting all of the metrics in this error message, can we just output the metrics that aren't in `all_scorers`?

---

that's a good point, 
what about if user specify 1 valid and 1 invalid metric? should I fail or as implemented now - we fail only if all metrics are invalid?

---

Should fail if at least one metric is invalid